Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: assets dep clean test gotest gotestrace jstest run run-dev run-hmr ctags
 
-VERSION ?= $(shell git describe --always --tags)
+VERSION = 1.7.0
```

Comment:
normally i'd say let's leave the automation, or otherwise update our release process docs, but considering the circumstance this seems ok.

Reply:
Sadly this doesn't work based on our process since we tag off the release branch. This is why master has been out of date since april.

Reply:
word